payloads/library/general/BunnyAlerts/payload.txt

```diff
@@ -0,0 +1,34 @@
+# Title:         Bunny Alerts
+# Description:   Sends a message to Slack channel, and writes message in notepad
+# Author:        Jesse Allen
+# Version:       1.0
+# Category:      General
+# Attackmodes:   HID
+# Target:        Windows 10, 11
+
+LED SETUP
+
+GET SWITCH_POSITION
+
+ATTACKMODE HID STORAGE
+
+LED STAGE1
+QUACK DELAY 1000
+QUACK GUI r
+QUACK DELAY 1000
+
+LED STAGE2
+# run PowerShell script
+
+QUACK STRING powershell -NoP -NonI -W Hidden ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\alert.ps1')"

```

Comment:
I would recommend using a variable assignment for the drive label as not everyone's configuration will have the default `BashBunny`  drive label. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
QUACK STRING powershell -NoP -NonI -W Hidden ".((gwmi win32_volume -f 'label=''$DRIVE_LABEL''').Name+'payloads\\$SWITCH_POSITION\alert.ps1')"
```